src/server/test/completion.spec.ts

```diff
@@ -113,7 +113,7 @@ describe('completion', () => {
         );
 
         const expected = [
-            { label: 'new', detail: 'a.mo', insertText: 'new', kind: 3 },
+            { label: 'new', detail: 'simple.mo', insertText: 'new', kind: 3 },
```

Comment:
These are from `test/completion` directory, so different `a.mo`s, and shouldn't be changed.

Reply:
Fixed; thanks for pointing that out.